code/modules/reagents/chemistry/recipes/food.dm

```diff
@@ -197,4 +197,14 @@
 	required_reagents = list("beff" = 1, "saltpetre" = 1, "synthflesh" = 1)
 	result_amount = 2
 	mix_message = "The beff and the synthflesh combine to form a smoky red log."
-	mix_sound = 'sound/effects/blobattack.ogg'
\ No newline at end of file
+	mix_sound = 'sound/effects/blobattack.ogg'
+
+/datum/chemical_reaction/enzyme
+	name = "Universal enzyme"
+	id = "enzyme"
+	result = "enzyme"
+	required_reagents = list("plantmatter" = 1, "sugar" = 1)
+	result_amount = 2
+	min_temp = 750
+	mix_message = "The mixture bubbles and leaves behind the enzymes."
```

Comment:
The only thing I would do is make this sound a bit nicer. I'm not sure what I'd write though.

Reply:
Yeah. How the hell would you ' see ' an enzyme with your naked eye, anyway? I'll think about it. But this isn't that important, it can always be changed, even after it is - perhaps - merged.